Sort by rbd (#163)
DFE-Digital/bat-design-history: app/posts/manage-teacher-training-applications/2020-06-10-sorting-by-rbd-date.md

---

> title: Sorting by reject by default date

Reviewer: ```suggestion
title: Sorting by reject by default (RBD) date
```

description: Let users sort by RBD date
date: 2020-06-03
tags:
- MN014
---

{% from "figure/macro.njk" import appFigure with context %}

## User needs

{% from "user-needs/macro.njk" import appUserNeeds %}
{{ appUserNeeds({ items: collections['user-need'] | slugs(tags)}) }}

## How it works

> By default the application list is sorted by last changed which means that the application has had a change of status of a note added to it.

Reviewer: ```suggestion
By default the application list is sorted by last changed, meaning the application’s status recently change or a note was added to it.
```

The user can now choose to sort by RBD date (days left to respond until the application is rejected automatically).

When the user sorts by RBD date the days left to respond replaces the last changed date for each application in the list that has a state of submitted.

## Design rationale

> The sort controls are aligned left to keep it located in the same place on small screens (where the filter toggle button appears on the right) and also gives it the best chance of being seen by users who zoom.

Author: ```suggestion
The sort controls are left aligned so that is consistently located on small screens (where the filter toggle button appears right aligned) and also gives it the best chance of being seen by users who zoom.
```

Reviewer: "so that it is"
